stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -775,6 +779,35 @@ public func withUnsafeBytes<T, Result>(
   }
 }
 
+/// Invokes the given closure with a buffer pointer covering the raw bytes of
+/// the given argument.
+///
+/// The buffer pointer argument to the `body` closure provides a collection
+/// interface to the raw bytes of `arg`. The buffer is the size of the
+/// instance passed as `arg` and does not include any remote storage.
+///
+/// - Parameters:
+///   - arg: An instance to temporarily access through a raw buffer pointer.
```

Comment:
This parameter is named `value`, and we've got lots of `arg`s above—what do you think of standardizing on `value` throughout? It's less j`arg`ony.